packages/core/src/types.ts

```diff
@@ -396,6 +396,12 @@ export interface ConfigBase<Theme extends {} = {}> {
    * @default false
    */
   details?: boolean
+
+  /**
+   * Whether to use the top-level await
+   * @default true
+   */
+  topLevelAwait?: boolean
```

Comment:
It should be in Vite plugin's config. And maybe `hmrTopLevelAwait`?

Reply:
This is not latest code, i have moved.